framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -785,6 +822,17 @@ protected void processCommonAssignment(TransferInput<V, S> in, Node lhs,
         // add new information based on conditional postcondition
         processConditionalPostconditions(n, method, tree, thenStore, elseStore);
 
+        if (inferSignatures) {
+            // Updates the inferred parameter type of the invoked method
+            ClassTree classTree = analysis.getContainingClass(n.getTree());
+            if (classTree != null) { // If method is invoked from a static context, classTree == null.
```

Comment:
It seems there is no signature inference for static methods.  Is that true?  If so, why?